packages/contracts-bedrock/scripts/DeployImplementations.s.sol

```diff
@@ -550,7 +550,7 @@ contract DeployImplementations is Script {
         vm.broadcast(msg.sender);
         IProxy proxy = IProxy(
             DeployUtils.create1({
-                _name: "Proxy",
+                _name: "forge-artifacts/Proxy.sol/Proxy.json",
```

Comment:
This implies that the proxy is still being compiled with the older version of solc?

Reply:
Wondering if there is a way around this somehow

Reply:
Nope, everything uses 0.8.25. But using `Proxy.sol:Proxy` like before, the test fails saying it's not unique... Is there a better solution to this?

Reply:
Perhaps `forge clean` fixes this?

Reply:
It's coming from `lib/openzeppelin-contracts-v5/contracts/proxy/Proxy.sol`

Reply:
Which itself is coming from `lib/openzeppelin-contracts-v5/contracts/proxy/beacon/BeaconProxy.sol`

Reply:
rip, good catch. This is an example as to why reducing external deps makes things a bit harder. I could be ok with it as is although I do not love it. I would prefer to have our own beacon proxy impl that doesn't use the crazy inheritance of oz

Reply:
Perhaps the following syntax will work:

```
bytes memory code = vm.getDeployedCode("universal/Proxy.sol:Proxy");
```

taken from https://github.com/ethereum-optimism/optimism/pull/12321

Reply:
That's how it was used in some places which i had to change to `forge-artifacts/Proxy.sol/Proxy.json` because it was not working

Reply:
ruh roh, no it looks like the failure is not in solidity but in the go implementation of `vm.getCode()`... :/